config/common.sh

```diff
@@ -33,6 +33,9 @@ export DM_ADMIN_MANAGER_USER_PASSWORD=${DM_ADMIN_MANAGER_USER_PASSWORD:=Password
 export DM_ADMIN_FRAMEWORK_MANAGER_USER_EMAIL=${DM_ADMIN_FRAMEWORK_MANAGER_USER_EMAIL:=functional-tests-default-admin-framework-manager-user@user.marketplace.team}
 export DM_ADMIN_FRAMEWORK_MANAGER_USER_PASSWORD=${DM_ADMIN_FRAMEWORK_MANAGER_USER_PASSWORD:=Password1234}
 
+export DM_ADMIN_CCS_DATA_CONTROLLER_USER_EMAIL=${DM_ADMIN_CCS_DATA_CONTROLLER_USER_EMAIL:=admin-ccs-data-controller-user@user.marketplace.team}
```

Comment:
Are you happy to change the email to be similar to the others? I realise it is a bit verbose...

```suggestion
export DM_ADMIN_CCS_DATA_CONTROLLER_USER_EMAIL=${DM_ADMIN_CCS_DATA_CONTROLLER_USER_EMAIL:=functional-tests-default-admin-ccs-data-controller-user@user.marketplace.team}
```

Reply:
Oh yes, didn't spot that. Will change!